services/app-api/src/resolvers/user/updateCMSUser.test.ts

```diff
@@ -97,6 +98,112 @@ describe('updateCMSUser', () => {
         )
     })
 
+    it('changes CMS users division assignment and creates an audit log', async () => {
+        const firstDivisionAssignment: Division = 'DMCO'
+        const secondDivisionAssignment: Division = 'DMCP'
+
+        const testAdminUser: UserType = {
+            id: 'd60e82de-13d7-459b-825e-61ce6ca2eb36',
+            role: 'ADMIN_USER',
+            email: 'iroh@example.com',
+            familyName: 'Iroh',
+            givenName: 'Uncle',
+        }
+
+        const prismaClient = await sharedTestPrismaClient()
+        const postgresStore = NewPostgresStore(prismaClient)
+        const server = await constructTestPostgresServer({
+            store: postgresStore,
+            context: {
+                user: testAdminUser,
+            },
+        })
+
+        const cmsUserID = uuidv4()
+        const userToInsert: InsertUserArgsType = {
+            userID: cmsUserID,
+            role: 'CMS_USER',
+            givenName: 'Zuko',
+            familyName: 'Firebender',
+            email: 'zuko@example.com',
+        }
+
+        const newUser = await postgresStore.insertUser(userToInsert)
+        if (isStoreError(newUser)) {
+            throw new Error(newUser.code)
+        }
+
+        // make the first update to the division assignment
+        const firstUpdateRes = await server.executeOperation({
+            query: UPDATE_CMS_USER,
+            variables: {
+                input: {
+                    cmsUserID: cmsUserID,
+                    divisionAssignment: firstDivisionAssignment,
+                },
+            },
+        })
+
+        expect(firstUpdateRes.data).toBeDefined()
+        expect(firstUpdateRes.errors).toBeUndefined()
+
+        if (!firstUpdateRes.data) {
+            throw new Error('no data')
+        }
+
+        const firstUpdateToUser = firstUpdateRes.data.updateCMSUser.user
+        expect(firstUpdateToUser.email).toBe(newUser.email)
+        // division assignment should now be set
+        expect(firstUpdateToUser.divisionAssignment).toBe(
+            firstDivisionAssignment
+        )
+
+        // make the second update to the division assignment
+        const secondUpdateRes = await server.executeOperation({
+            query: UPDATE_CMS_USER,
+            variables: {
+                input: {
+                    cmsUserID: cmsUserID,
+                    divisionAssignment: secondDivisionAssignment,
+                },
+            },
+        })
+
+        expect(secondUpdateRes.data).toBeDefined()
+        expect(secondUpdateRes.errors).toBeUndefined()
+
+        if (!secondUpdateRes.data) {
+            throw new Error('no data')
+        }
+
+        const secondUpdateToUser = secondUpdateRes.data.updateCMSUser.user
+        expect(secondUpdateToUser.email).toBe(newUser.email)
+        expect(secondUpdateToUser.divisionAssignment).toBe(
+            secondDivisionAssignment
+        )
+
+        const auditLogs = await prismaClient.userAudit.findMany({
```

Comment:
nice, I think this is totally appropriate for a test to check the db directly for something we haven't exposed anywhere.